app/public/js/controllers/activity.js

```diff
@@ -18,7 +18,7 @@
 
```

Comment:
If we use Activities controller now, the old Activity controller has to dissapear